examples/mnist_denoising_autoencoder.py

```diff
@@ -28,6 +28,9 @@
 from keras.datasets import mnist
 import numpy as np
 import matplotlib.pyplot as plt
+from PIL import Image
```

Comment:
You could do that with Matplotlib, to avoid a new dependency

Reply:
Couldn't figure out how to remove all the whitespace around. PIL is used in the examples/ already.
https://user-images.githubusercontent.com/14060629/34853831-c931deec-f6ea-11e7-8d43-8093eb0a3705.png

Reply:
I understand that it's better to do the image stacking and formatting in Numpy, to avoid whitespace and so on. But you're only using PIL to convert your Numpy array to an image, and I'm saying you can use Matplotlib to do this step, e.g. `plt.imshow(imgs, cmap='gray')

Reply:
Using Matplotlib to save the image leaves whitespace around it. I couldn't figure out how to remove it.  `bbox_inches='tight', pad_inches=0, plt.tight_layout()` didn't work.

Compare matplotlib (whitespace)
https://user-images.githubusercontent.com/14060629/34853831-c931deec-f6ea-11e7-8d43-8093eb0a3705.png

to PIL (no whitespace)
https://user-images.githubusercontent.com/14060629/34853907-31da9830-f6eb-11e7-8da5-5756dee2610a.png